example/example.dart

```diff
@@ -99,5 +99,27 @@ main() async {
 
   var relatedArtists = await spotify.artists.relatedArtists('0OdUWJ0sBjDrqHygGUXeCF');
   print('related Artists: ${relatedArtists.length}');
+
```

Comment:
Please remove that change from the PR, since it's coming in with #28 :smile_cat: